src/main/scala/gestalt/Toolbox.scala

```diff
@@ -2,253 +2,226 @@ package scala.gestalt
 
 import scala.collection.immutable.Seq
 
-trait Toolbox { t =>
-  // portable trees -- minimum assumptions
-  type Tree <: { def tpe: Type } // TODO: structural types performance penalty.
-  type TypeTree <: Tree          // safety by construction -- implementation can have TypeTree = Tree
-  type Type
+trait Modifiers[T] {
```

Comment:
Also can we have the trait Modifiers inside the Toolbox?
Then it will not have to have the type parameter `T` - annotations then are just trees.
Modifiers do not make sense outside the context of a Toolbox. For example, modifiers with annotations of type `String` do not make sense.